src/main/java/com/farao_community/farao/gridcapa/export/adapter/ClientAdapter.java

```
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa.export.adapter;
+
+import java.io.InputStream;
+
+/**
+ * @author Oualid Aloui {@literal <oualid.aloui at rte-france.com>}
+ */
+
+public interface ClientAdapter {
+
+    void open() throws Exception;
```

Why use the most general `Exception` here ? It would be better to throw a custom exception and wrap the `IOException`, `JSchException` or `SftpException` in it in your implementations of the interface, in order make it generic but not to much.